overlays/nixpkgs.nix

```diff
@@ -15,4 +15,14 @@ final: prev: {
   azure-cli = prev.azure-cli.override {
     withExtensions = with final.azure-cli.extensions; [ aks-preview ];
   };
+
+  erofs-utils = prev.erofs-utils.overrideAttrs (
```

Comment:
We might want to add a comment here with the PR description. This way it'd be more visible than in the PR / commit. But no strong opinion